lib/net.js

```diff
@@ -372,8 +372,8 @@ function Socket(options) {
   this[kBytesRead] = 0;
   this[kBytesWritten] = 0;
 }
-Object.setPrototypeOf(Socket.prototype, stream.Duplex.prototype);
-Object.setPrototypeOf(Socket, stream.Duplex);
+setPrototypeOf(Socket.prototype, stream.Duplex.prototype);
```

Comment:
Could you name these to be a bit more explicit like `objectSetPrototypeOf`, `objectDefineProperty` (like here for example https://github.com/nodejs/node/pull/30246/files)

Reply:
makes sense, done!

Reply:
that construction still looks a bit odd, though. is it usual to link to the relevant issue in this case?

Reply:
> that construction still looks a bit odd, though. is it usual to link to the relevant issue in this case?

Are you referring to destructuring all properties and using them directly? If that's the case then yeah we do reference relevant issues usually but in this case, it'd just flood the code with the same comments so it's not needed here.